ex1_student_solution.py

```diff
@@ -106,9 +115,19 @@ def compute_forward_homography_fast(
         Returns:
             The forward homography of the source image to its destination.
         """
-        # return new_image
-        """INSERT YOUR CODE HERE"""
-        pass
+        dst_image = np.zeros(dst_image_shape, dtype=np.uint8)
+        h, w = src_image.shape[0], src_image.shape[1]
```

Comment:
Just so you know you can write: h,w = src_image.shape
it will unpack the tuple by itself

Reply:
looks better, will change